src/borg/archiver.py

```diff
@@ -2387,8 +2388,8 @@ def build_parser(self, prog=None):
 
         When rechunking space usage can be substantial, expect at least the entire
         deduplicated size of the archives using the previous chunker params.
-        When recompressing approximately 1 % of the repository size or 512 MB
-        (whichever is greater) of additional space is used.
+        When recompressing expect approx throughput / checkpoint-interval in space usage,
```

Comment:
add a dot when abbreviating.
also maybe (t / c-i) makes it clearer that it is meant as a computation.